notice/1026/2013-22752_20140101.xml

```diff
@@ -329,7 +264,7 @@
                 </paragraph>
                 </change><change operation="modified" label="1026-5-a-1-ii-B"><paragraph label="1026-5-a-1-ii-B" marker="(B)">
-                    <content>The following disclosures need not be in a retainable form: Disclosures that need not be written under paragraph <ref target="1026-5-a-1-ii-A" reftype="internal">(a)(1)(ii)(A)</ref> of this section; disclosures for <ref target="1026-2-a-14" reftype="term">credit</ref> and <ref target="1026-2-a-15-iii" reftype="term">charge card</ref> applications and <ref target="1026-46-d-1-i" reftype="term">solicitations</ref><ref target="1026-60-a-1" reftype="term">solicitations</ref> under § <ref target="1026-60" reftype="internal">1026.60</ref>; home-equity disclosures under § <ref target="1026-40-d" reftype="internal">1026.40(d)</ref>; the alternative summary billing-rights statement under § <ref target="1026-9-a-2" reftype="internal">1026.9(a)(2)</ref>; the <ref target="1026-2-a-14" reftype="term">credit</ref> and <ref target="1026-2-a-15-iii" reftype="term">charge card</ref> renewal disclosures required under § <ref target="1026-9-e" reftype="internal">1026.9(e)</ref>; and the payment requirements under § <ref target="1026-10-b" reftype="internal">1026.10(b)</ref>, except as provided in § <ref target="1026-7-b-13" reftype="internal">1026.7(b)(13)</ref>.</content>
+                    <content>The following disclosures need not be in a retainable form: Disclosures that need not be written under paragraph <ref target="1026-5-a-1-ii-A" reftype="internal">(a)(1)(ii)(A)</ref> of this section; disclosures for <ref target="1026-2-a-14" reftype="term">credit</ref> and <ref target="1026-2-a-15-iii" reftype="term">charge card</ref> applications and <ref target="1026-46-d-1-i" reftype="term">solicitations</ref> <ref target="1026-60-a-1" reftype="term">solicitations</ref> under § <ref target="1026-60" reftype="internal">1026.60</ref>; home-equity disclosures under § <ref target="1026-40-d" reftype="internal">1026.40(d)</ref>; the alternative summary billing-rights statement under § <ref target="1026-9-a-2" reftype="internal">1026.9(a)(2)</ref>; the <ref target="1026-2-a-14" reftype="term">credit</ref> and <ref target="1026-2-a-15-iii" reftype="term">charge card</ref> renewal disclosures required under § <ref target="1026-9-e" reftype="internal">1026.9(e)</ref>; and the payment requirements under § <ref target="1026-10-b" reftype="internal">1026.10(b)</ref>, except as provided in § <ref target="1026-7-b-13" reftype="internal">1026.7(b)(13)</ref>.</content>
```

Comment:
Does this change actually fix the error mentioned in GHE#1992? It seems like there'd still be two adjacent "solicitations" links here.